### DISCUSSION TITLE
rownames of a ArchRProject

### DISCUSSION THREAD

**Discussion by sunnyyakima** [GreenleafLab/ArchR#1820] [Questions / Documentation]
I am trying to reproduce a piece of codes from publication.

they used followings:
sc <- ArchRProject(ArrowFiles = ArrowFiles,
    outputDirectory = "snATACseq/clustering", copyArrows = TRUE)

tss_outliers_names[[sample_i]] <-
        rownames(sc)[sc$Sample==sample_i][tss_outliers[[sample_i]]]

got an error, it seems that there is no such rownames(ArchrProject) ?

**Comment by rcorces** [comment]
I dont believe that calling `rownames()` of a project was ever supported.
